### ISSUE TITLE
Using the x option still gives me 'openBinaryFile: does not exist'

### ISSUE THREAD

**Issue by njgte** [ludeeus/action-shellcheck#49]
## Describe the issue

I am using shellcheck on a bash script which sources another file using a relative path like 

```shell
# shellcheck source=./local_file
source "${BASH_SOURCE%/*}/local_file"
```

when I run shellcheck -x on my machine this doesn't give errors. 

I am using action-shellcheck to run shellcheck on push. 

My action file looks like this:

```yml
name: shellcheck
on: [push] ## run on every push
jobs:
  shellcheck:
    name: Shellcheck
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v2
    - name: Run ShellCheck
      uses: ludeeus/action-shellcheck@master
      env:
        SHELLCHECK_OPTS: -x
```

However the action fails and I get the error `note: Not following: ./prod.config: openBinaryFile: does not exist (No such file or directory) [SC1091]`.
This is the same error that I get when I run shellcheck locally without the -x option.

**Comment by ludeeus** [comment]
That message means that the file does not exist.
If you remove the "-x" from `SHELLCHECK_OPTS` you will get another message.

**Comment by njgte** [comment]
Thank you for your reply @ludeeus. I discovered that the reason for my problem was the directory that shellcheck is run under. I could solve the problem by specifying a scripts directory using the -P option.
